### DISCUSSION TITLE
pinned column persists after restart

### DISCUSSION THREAD

**Discussion by DavidHJong** [dbeaver/dbeaver#29288] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
When re-open a table/view, or start the dbeaver, the pinned columns are no longer pinned. Many a time we need some columns to be persistently pinned, so I hope there would be a feature that pinned columns will persist across sessions. 

**Describe the solution you'd like**
One way is the add an option in preferences to let users decide if they want pinned columns be persistent across sessions. I don't know how pinned columns are realized programmatically, but saving that state to a file could help persisting it across sessions. 

Does anyone have any thoughts?

**Comment by E1izabeth** [comment, marked as answer]
You can press this button and columns configuration will be saved
![image](https://github.com/dbeaver/dbeaver/assets/28875055/6a559401-d28d-4123-bb07-d27f9946bb5e)

**Comment by DavidHJong** [comment]
Thank you! It works.
